adr/2020_11_18_sending_emails_via_api.md

```diff
@@ -0,0 +1,43 @@
+Date: 2020-11-18
+
+# API - Sending emails
+
+In power from version: master (?)
+
+Related PRs: #11754, #12050
+
+### Problem to solve
+
+How to send email using api
+
+### Context
+
+With more and more api coverage we had to decide how we should handle sending emails.
```

Comment:
some operations in Sylius as the final step send emails to the customer, for full API coverage we need to handle it also by API

Reply:
or sth like this :D